bitnami/minio/values.yaml

```diff
@@ -746,6 +746,14 @@ networkPolicy:
   ## listening on. When true, MinIO&reg; will accept connections from any source (with the correct destination port).
   ##
   allowExternal: true
+  
```

Comment:
Hi, 
  Could you take a look to the linter message ?

Reply:
Hi
I don't know how to fix
`Error: [trailing-spaces] trailing spaces`

How do I run the lint?

Reply:
Ok found the error, I pushed the fix and the checks are now passing